app/ts/library/constants.ts

```diff
@@ -135,3 +135,5 @@ export function getAssetMetadata(chainId: string, address?: string) {
 	if (match === undefined) throw new Error('Specified asset is unsupported')
 	return match
 }
+
+export const STORAGE_KEY_RECENTS = 'txns'
```

Comment:
this could have a more verbose name. Eg "sentTransactions"

Reply:
`txns` is only used as storage key, or did you mean `STORAGE_KEY_RECENTS`?